doc/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.fhir.analytics</groupId>
+    <artifactId>root</artifactId>
+    <version>0.2.7-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>doc</artifactId>
+  <packaging>pom</packaging>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
```

Comment:
Is it possible to move this `<plugin>` to the `<build>` section of the root `pom.xml` such that we do not need to repeat this segment in all child `pom.xml` files?

Reply:
Noted. Initially I thought we might need project level configurations especially for the projects that have python source code, but now that we've gone a different route we don't need this here.